Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -989,7 +957,7 @@ class BrowserViewController: UIViewController {
         profile.bookmarks.modelFactory >>== {
             $0.isBookmarked(url).uponQueue(DispatchQueue.main) { [weak tab] result in
                 guard let bookmarked = result.successValue else {
-                    log.error("Error getting bookmark status: \(result.failureValue ??? "nil").")
+                    print("Error getting bookmark status: \(result.failureValue ??? "nil").")
```

Comment:
curious why you decided to switch it to print?

Reply:
I removed XCGLogger dependency for the class. XCGLogger is overkill except for sync and share debugging IMO.